### PR TITLE
Document RHEL and Clones, add Rocky Linux 8 and PostgreSQL to the matrix

### PR DESCRIPTION
**Pull request by juliogonzalez**
Fixes tds-fdw/tds_fdw#265 and tds-fdw/tds_fdw#297 

The reasoning for the renaming of `InstallCentOS.md` is simple: we can't add CentOS 8, as it goes EoL by the end of this year, and by using the RHEL name, we can write common instructions for 7 and 8, that then can be used for every other maintained clone, such as CentOS or Oracle Linux 7, Rocky Linux or AlmaLinux 8, and many others.

The markdown files are better reviewed as a whole, instead of using the diff.

Needless to  say, the badges for Rocky Linux 8 won't work until we merge this PR and the CI runs, but https://jenkins.juliogonzalez.es/job/tds_fdw-build-free/31/ was able to run the tests for Rocky Linux 8 without any problem).

- [ ] Reenable https://jenkins.juliogonzalez.es/job/tds_fdw-build/ as soon as this PR is merged

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@ sql/*--*.sql
 *.o
 *.bc
 *.so
+tests/lib/__pycache__
```
Comment (juliogonzalez) [inline .gitignore:7]: So we don't include pycache, as it happened to me today :-)

File: tests/lib/tests.py [hunk 2]
```diff
@@ -98,7 +98,7 @@ def get_logs_path(conn, dbtype):
         pids = [pid for pid in listdir('/proc') if pid.isdigit()]
         for pid in pids:
             try:
-                cmdline = open('/proc/' + pid + '/cmdline', 'rb').read()
+                cmdline = open('/proc/' + pid + '/cmdline', 'r').read()
```
Comment (juliogonzalez) [inline tests/lib/tests.py:101]: I don't know why this worked at Ubuntu 20.04 (Python 3.8), but it doesn't work at Rocky Linux (Python 3.6 by default). Anyway the file behaves as text, not binary, and this way it works for Python 2 on CentOS7 as well.

**Comment by juliogonzalez** [comment]
It seems I am putting way too much load on the Jenkins worker #2, and they are not able to download an extract all the images, as they stay for minutes at:
```
21:16:51 698d50db42d0: Pull complete
21:16:53 8ce6ff6761d7: Pull complete
21:16:55 182e31163e59: Verifying Checksum
21:16:55 182e31163e59: Download complete
```
I will decrease the number of executors there, to see if the problem gets finally fixed.

**Comment by juliogonzalez** [comment]
Retest this, please

**Comment by juliogonzalez** [comment]
All yours @GeoffMontee

I'd merge it on my own, but since we I am changing the doc, I'd prefer to have your approval :-)

**Comment by GeoffMontee** [comment]
Thanks for all your work on this, @juliogonzalez! This PR has been merged.

**Pull request merged**